README.md

```diff
@@ -1,5 +1,7 @@
 # loopback-next
 
+![LoopBack4 logo](https://raw.githubusercontent.com/strongloop/loopback.io/gh-pages/images/branding/logo/blue/loopback-sm.png)
+
```

Comment:
The logo is too big.

Reply:
Maybe the logo should come after other badges?

Reply:
I can make the change.. 
I was getting inspirations from the github repos of Node and Express, which put their logo at the very top  :) 
https://github.com/expressjs/express
https://github.com/nodejs/node

Reply:
Node does not have badges.

Reply:
We can remove `loopback-next` title to match `Express` style.